site/_posts/2017-01-03-last-week-in-manageiq-a-brand-new-euwe.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Last Two Weeks in ManageIQ: A brand new Euwe"
+author: Julian Cheal
+date: 2017-01-03
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+As the end of 2016 grew closer, and the Winter Solstice brought with it either the longest night or shortest night, depending on which hemisphere you frequent, ManageIQ cut its latest release Euwe. You can download it [here][1] or clone it from Github [here][2] Happy New Year!
+
+## Featured
+
+Over the last two weeks amongst all the New Year's celebrations there were [70 pull requests merged][3]! These pull requests (PRs) were comprised of [229 commits][4]. Why not start your New Year by contributing to Open Source, just check out our [issue tracker][5] to see something that tickles your fancy, or join our chat on [Gitter][6] to find out more.
+
+## Improved
+
+If removing technical debt is considered an improvement (answer: it should be), then these last two weeks won't disappoint.
+First up is [Tim Wade][7] removing some [dead paths][8] in `MiqExpression.operands2rubyvalue`. Next up is [Tim Wade][7] again! This time [fixing invalid chargeback factories][9] in our test suit.
+
+And finally, a useful improvement from [Ari Zellner][10] Changing `Images` to `ContainerImages` in our [chargeback rate assignments][11].
+
+## Fixed
+
+After a normally expensive December, people are often frugal in January; which is good as that's when the sales start! And who wants to overpay when you can get something for less. That's why [Šimon Lukašík][12] fixed a bug in [chargeback reporting][13]
+
+[Laura Galis][14] Fixed the case where an empty link was being created if an [orchestration stack was empty][15].
+
+I know, I know it's January and we're all still zoned out from the holidays. Don't despair [Gregg Tanzillo][16] has a solution, he's fixed [zone scoping][17] for a server's region!
+
+## New
+
+With new code comes new bugs, one of our new projects is creating a new physical infrastructure provider, this is still ongoing work. However that has led to a couple of bugs, but [Dávid Halász][18] to the rescue he's fixed our [Rubocop][19] [style guide][20] to [not complain][21] about the `$lenovo_log` global; the logging variable used by our first physical infrastructure provider, Lenovo.
```

Comment:
"our `[Rubocop][19]`" - The link works, but I found that I was expecting to see something tied to ManageIQ.  - I'm not sure I know what else to put here though,  as the style guide url clearly points to our style guide.  

I guess this is just a note of interest as viewed by a 'casual' reader.  😈